### PR TITLE
[CLAIM] Fixes consumed claims check for other chains

### PR DESCRIPTION
**Pull request by W3stside**
Merges into #2313 

Now checks if consumed using hooks

### CODE REVIEW

File: src/custom/state/claim/hooks/index.ts [hunk 1]
```diff
@@ -139,8 +139,8 @@ type ClassifiedUserClaims = {
  *
  * @param account
  */
-export function useClassifiedUserClaims(account: Account): ClassifiedUserClaims {
-  const userClaims = useUserClaims(account)
+export function useClassifiedUserClaims(account: Account, optionalChainId?: number): ClassifiedUserClaims {
```
Comment (alfetopito) [inline src/custom/state/claim/hooks/index.ts:142]: Can you use SupportedChainId as type instead?

**Review by alfetopito — APPROVED** [review]
Awesome!

Only a tiny suggestion

**Comment by W3stside** [comment]
post merge review pls

**Comment by anxolin** [comment]
Approve!

**Pull request merged**